games/u.yaml

```
@@ -50,10 +50,10 @@
   - 'X-COM: Terror from the Deep'
   - 'X-COM: Apocalypse'
   - 'UFO: Enemy Unknown'
-  repo: https://sourceforge.net/projects/ufo2000/
+  repo: https://github.com/ufo2000/ufo2000
```

```suggestion
  repo: https://sourceforge.net/projects/ufo2000/
```
I don't think this should be changed; the github description says it is a repo mirror. The project download page still links to sourceforge

---

Done